README.md

```diff
@@ -22,6 +22,18 @@ The database access parameters are stored in `tests/config.ini`.
 
 ### Solr Server
 
+### Directory 
+
+The workspace directory for testing can be created by using 
+
+    ant prepare-workspace
+    
+### Configuration
+
+The config file (config.ini) for the testing configurations in the framework can be created using
+
+    ant prepare-config
```

Comment:
No,  not really. This creates a default configuration that will work on our CI system, but it probably isn't enough information for a developer trying to setup testing on his own system.

Reply:
What happened to this comment?

Reply:
I moved this section up before Database section. Because then it can be referred to edit config file.